### PR TITLE
Change LICENCE to LICENSE

### PR DESCRIPTION
**Pull request by MaxAltena**
I fixed a type made in some files regarding the license.

### CODE REVIEW

**Comment by natterstefan** [comment]
Hello @MaxAltena, thanks for your contribution. I just saw you created your own config. Awesome. Especially, [the annotations in your `prettier.config.js`](https://github.com/MaxAltena/eslint-config-maxaltena/commit/ed26590152ef9a62f642fc8e470713581baa7da4#diff-ab9b36d6b3d8ecb76af4ab93254abde4) are great. I might copy them into my file as well.

Regarding the typo thing. It depens whom you ask. ["Licence" is the British English noun and "license" the American English one](https://www.grammarly.com/blog/licence-license/).

Still, I like the idea to be consistent with the `license` in [package.json](https://github.com/natterstefan/eslint-config-ns/blob/master/package.json#L6).

Thanks for taking care of it!

File: README.md [hunk 1]
```diff
@@ -92,9 +92,9 @@ This ESLint configuration comes with some fundamental assumptions:
 Even though I made some assumptions, [you can easily overwrite, extend and unset
 rules and any other setting in your custom eslint config](https://eslint.org/docs/user-guide/configuring).
 
-## LICENCE
+## LICENSE
 
-[MIT](LICENCE)
+[MIT](LICENSE)
```
Comment (natterstefan) [inline README.md]: Would you mind updating the badge as well?

```md
[![GitHub license](https://img.shields.io/github/license/natterstefan/eslint-config-ns.svg)](https://github.com/natterstefan/eslint-config-ns/blob/master/LICENSE
```

That would be highly appreciated, thanks!

**Comment by MaxAltena** [comment]
No problem @natterstefan!

**Comment by natterstefan** [comment]
Thank you very much @MaxAltena. :)

**Pull request merged**